src/applications/vaos/appointment-list/components/cancel/CancelAppointmentFailedModal.jsx

```diff
@@ -34,7 +34,7 @@ export default function CancelAppointmentFailedModal({
           contact your medical center to cancel:
         </p>
       )}
-      <p>
+      <div>
```

Comment:
Why the change?

Reply:
Changing the <p> tag to <div> fixes the warning message that <h5> cannot be a descent of a paragraph tag. The <h5> tag is generated from `FacilityAddress` function which is nested inside the now updated <div> tag .